pkg/server/message.go

```diff
@@ -254,69 +262,54 @@ func (t *TranslateServiceServer) UpdateMessages(
 
 // helpers
 
-// populateTranslatedMessages adds any missing messages from the original messages to all translated messages.
-func (t *TranslateServiceServer) populateTranslatedMessages(
+// fuzzyTranslate fuzzy translates any untranslated messages,
+// returns messagesSlice containing refreshed translations.
+func (t *TranslateServiceServer) fuzzyTranslate(
```

Comment:
I am thinking about removing this helper and moving its logic to `fuzzy` pkg, but once again it requires many changes...

It would be more appropriate that Translate takes `[]Messages`, and changes in place all values, instead of returning new `Messages`

Something like
```go
Translate(ctx context.Context, messages ...model.Messages) error
// or
Translate(ctx context.Context, messages model.MessagesSlice) error
```

Reply:
Scope creep, leave it for another PR. We need to finish this PR. It's taking too long.